src/main.cpp

```diff
@@ -214,15 +244,15 @@ bool influxDbIsConfigured(){
 /**
  * @influxDbParseFields:
  *
- * Supported: 
+ * Supported:
  * "id","pm1","pm25","pm10,"hum","tmp","lat","lng","alt","spd","stime","tstp"
- * 
+ *
  */
 void influxDbParseFields(char* fields){
   sprintf(
     fields,
-    "pm1=%u,pm25=%u,pm10=%u,hum=%d,tmp=%d,lat=%d,lng=%d,alt=%d,spd=%d,stime=%i,tstp=%u",
-    0,apm25,apm10,0,0,0,0,0,0,stime,0
+    "pm1=%u,pm25=%u,pm10=%u,hum=%u,tmp=%u,lat=%d,lng=%d,alt=%d,spd=%d,stime=%i,tstp=%u", //cambie de %d a %u para hum y tmp
```

Comment:
Only comments for the code and in english please

Reply:
Removed